modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java

```diff
@@ -155,7 +156,9 @@ class Item {
           Jsons.p("title", title),
           Jsons.p("partOf", event.getDublinCore().getFirst(DublinCore.PROPERTY_IS_PART_OF)),
           Jsons.p("description", event.getDublinCore().getFirst(PROPERTY_DESCRIPTION)),
-          Jsons.p("created", event.getDublinCore().getFirst(DublinCore.PROPERTY_CREATED)),
+          Jsons.p("created", Date.from(
+            Instant.parse(event.getDublinCore().getFirst(DublinCore.PROPERTY_CREATED))
+          ).getTime()),
```

Comment:
Shouldn't this be the same as

```suggestion
          Jsons.p("created", Instant.parse(
            event.getDublinCore().getFirst(DublinCore.PROPERTY_CREATED))
          ).getEpochSecond()),
```

without the detour via `Date`?